### PR TITLE
CNN/Dailymail Dataset Card

### PR DESCRIPTION
**Pull request by mcmillanmajora**
Link to the card page: https://github.com/mcmillanmajora/datasets/tree/cnn_dailymail_card/datasets/cnn_dailymail

One of the questions this dataset brings up is how we want to handle versioning of the cards to mirror versions of the dataset. The different versions of this dataset are used for different tasks (which may not be reflected in the versions that we currently have in the repo?), but it's only the structure that's changing rather than the content in this particular case, at least between versions 2.0.0 and 3.0.0.

### CODE REVIEW

File: datasets/cnn_dailymail/README.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+## Who is/was involved in the dataset use and creation? 
+### Who are the dataset curators? 
+
+The data was originally collected by Karl Moritz Hermann, Tomáš Kočiský, Edward Grefenstette, Lasse Espeholt, Will Kay, Mustafa Suleyman, and Phil Blunsom of Google DeepMind. Tomáš Kočiský and Phil Blunsom are also affiliated with the University of Oxford. They released scripts to collect and process the data into the question answering format. 
+
+Ramesh Nallapati, Bowen Zhou, Cicero dos Santos, and Bing Xiang of IMB Watson and Çağlar Gu̇lçehre of Université de Montréal modified Hermann et al's collection scripts to restore the data to a summary format. They also produced both anonymized and non-anonymized versions.
+
+The code for the non-anonymized version is made publicly available by Abigail See of Stanford University, Peter J. Liu of Google Brain and Christopher D. Manning of Stanford University at https://github.com/abisee/cnn-dailymail. The work at Stanford University was supported by the DARPA DEFT ProgramAFRL contract no. FA8750-13-2-0040.
```
Comment (sgugger) [inline datasets/cnn_dailymail/README.md:54]:
```suggestion
The code for the non-anonymized version is made publicly available by Abigail See of Stanford University, Peter J. Liu of Google Brain and Christopher D. Manning of Stanford University at <https://github.com/abisee/cnn-dailymail>. The work at Stanford University was supported by the DARPA DEFT ProgramAFRL contract no. FA8750-13-2-0040.

```
(To make the URL clickable)

File: datasets/cnn_dailymail/README.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+The data consists of news articles and highlight sentences. In the question answering setting of the data, the articles are used as the context and entities are hidden one at a time in the highlight sentences, producing Cloze style questions where the goal of the model is to correctly guess which entity in the context has been hidden in the highlight. In the summarization setting, the highlight sentences are concatenated to form a summary of the article. The CNN articles were written between April 2007 and April 2015. The Daily Mail articles were written between June 2010 and April 2015. 
+
+### How was the data collected?
+
+The code for the original data collection is available at https://github.com/deepmind/rc-data. The articles were downloaded using archives of www.cnn.com and www.dailymail.co.uk on the Wayback Machine. Articles were not included in the Version 1.0.0 collection if they exceeded 2000 tokens. Due to accessibility issues with the Wayback Machine, Kyunghyun Cho has made the datasets available at https://cs.nyu.edu/~kcho/DMQA/. An updated version of the code that does not anonymize the data is available at https://github.com/abisee/cnn-dailymail.
```
Comment (sgugger) [inline datasets/cnn_dailymail/README.md:70]:
```suggestion
The code for the original data collection is available at <https://github.com/deepmind/rc-data>. The articles were downloaded using archives of <www.cnn.com> and <www.dailymail.co.uk> on the Wayback Machine. Articles were not included in the Version 1.0.0 collection if they exceeded 2000 tokens. Due to accessibility issues with the Wayback Machine, Kyunghyun Cho has made the datasets available at <https://cs.nyu.edu/~kcho/DMQA/>. An updated version of the code that does not anonymize the data is available at <https://github.com/abisee/cnn-dailymail>. 

```

**Review by sgugger — APPROVED** [review]
Very cool!

**Review by lhoestq — APPROVED** [review]
Clear and well detailed, love it !

**Pull request merged**